CHANGELOG.md

```diff
@@ -7,6 +7,16 @@ be found.
 
 ## 1.13.0 (2016-12-08)
 
+**IMPORTANT**: In Docker 1.13 managed plugin api changed from the experimental version introduced in 
+Docker 1.12. Once upgrading to Docker 1.13 you might having issues at daemon start:
+> Error starting daemon: json: cannot unmarshal string into Go value of type types.PluginEnv
+
+To resolve this:
+
+- Remove plugins.json from: `/var/lib/docker/plugins/`
+- Restart Docker
+- Reinstall your plugins
```

Comment:
Do you think we should simplify and instruct people to uninstall any plugin before upgrading? (`docker plugin rm <plugin>`)